index.html

```diff
@@ -148,13 +148,13 @@ <h3 class="skills-text">Skills</h3>
         <div class="contact-dev">
           <h2 class="contactmeHeader">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
           <form method="post" autocomplete="off" action="https://formspree.io/f/xdovwjar" class="inputForm">
-            <input id="firstname-input" onchange="saveValue(this);" class="full_Name" type="text" placeholder="Full Name" maxlength="30" name="First-name" required />
-            <div class="fullName">
-              <input id="firstname-input" onchange="saveValue(this);" class="name" type="text" placeholder="First Name" maxlength="30" name="First-name" required />
-              <input id="secondname-input" onchange="saveValue(this);" class="name" type="text" placeholder="Second name" maxlength="30" name="Second-name" required />
-            </div>
+            <input id="fullName" onchange="saveValue(this);" class="full_Name" type="text" placeholder="Full Name" maxlength="30" name="First-name" required />
+            <!-- 
+            <input id="firstname-input" onchange="saveValue(this);" class="name" type="text" placeholder="First Name" maxlength="30" name="First-name" required />
+            <input id="secondname-input" onchange="saveValue(this);" class="name" type="text" placeholder="Second name" maxlength="30" name="Second-name" required /> -->
```

Comment:
Great job team 🤝

- [ ] Kindly remove commented code from your project as this is regarded as a bad practice.